games-roguelike/nethack/nethack-3.4.3-r5.ebuild

```diff
@@ -30,7 +30,7 @@ DEPEND="${RDEPEND}
 	)"
 
 BINDIR="/usr/games/bin"
-HACKDIR="/usr/share/games/${PN}"
+HACKDIR="/usr/lib/${PN}"
```

Comment:
Mh no... we should not install into this dir and it also causes my PM to throw warnings.
We might even remove the `lib` directory completely sometime in the future, so this has to be `HACKDIR="/usr/$(get_libdir)/${PN}"`, but I'd avoid `$(get_libdir)` in global scope, although HACKDIR is not part of metadata. Moving it to pkg_setup is probably cleaner and will still be available in all functions.